addon-test-support/-private/ember-exam-mocha-test-loader.js

```diff
@@ -48,8 +48,8 @@ export default class EmberExamMochaTestLoader extends TestLoader {
    * Loads the test modules depending on the urlParam
    */
   loadModules() {
-    const modulePathFilter = this._urlParams.get('modulePathFilter');
-    const testFilePathFilter = this._urlParams.get('testFilePathFilter');
+    const modulePath = this._urlParams.get('modulePath');
```

Comment:
is this a breaking change?

Reply:
We never released the previous PR into a version. So would that still be consider breaking?

Reply:
ah, nope then no breaking change.